port/linuxriscv64/omrsignal_context.c

```diff
@@ -73,11 +62,8 @@ infoForSignal(struct OMRPortLibrary *portLibrary, struct OMRUnixSignalInfo *info
 	case OMRPORT_SIG_SIGNAL_CODE:
 	case 3:
 		*name = "Signal_Code";
-		if( NULL != info->sigInfo) {
-			*value = &info->sigInfo->si_code;
-			return OMRPORT_SIG_VALUE_32;
-		}
-		break;
+		*value = &info->sigInfo->si_code;
```

Comment:
Why are the null checks not needed?  What do other platforms do?

Reply:
In theory we can add these null checks if the passed-in `gpinfo ` is null (actually it shouldn't be null once the signal is triggered)
e.g.
```
runtime\vm\gphandle.c
vmSignalHandler(struct J9PortLibrary* portLibrary, U_32 gpType, void* gpInfo, void* userData)
{
...
	crashData.gpInfo = gpInfo;
...
	j9sig_protect(
		writeCrashDataToConsole, &crashData,
		recursiveCrashHandler, &recursiveCrashData,
		J9PORT_SIG_FLAG_MAY_RETURN | J9PORT_SIG_FLAG_SIGALLSYNC,
		&result);
```
Beside, I didn't see the null check is added on any other platform in the existing implementation.

Reply:
Ok.  I'm fine with leaving it out then